granular_permissions.md

```
@@ -0,0 +1,163 @@
+# Base Permissions <ins>required</ins> for the Integration
+
+| OPNsense Permission | API Endpoints |
+| ----- | ----- |
+| Lobby: Dashboard | /api/diagnostics/system/systemInformation |
+| Status: Interfaces | /api/interfaces/overview/export |
+| System: Firmware | /api/core/firmware/status |
+
+# Granular Sync Permissions
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)